packages/jest-haste-map/src/types.ts

```diff
@@ -8,7 +8,10 @@
 import type {Stats} from 'graceful-fs';
 import type {Config} from '@jest/types';
 import type HasteFS from './HasteFS';
+// eslint-disable-next-line import/no-duplicates
```

Comment:
huh. can we do `import type ModuleMap, {IModuleMap} from './ModuleMap';`? If not, `import type {default as ModuleMap, IModuleMap} from './ModuleMap';`?